packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -108,7 +107,9 @@ class Plugin implements CLNPlugin {
         .toList();
     response["subscriptions"] = plugin.subscriptions;
     response["hooks"] = plugin.hooks.toList();
-    response["notifications"] = [];
+    response["notifications"] = plugin.notifications.values
+        .map((notification) => notification.toMap())
+        .toList();
     response["dynamic"] = plugin.dynamic;
```

Comment:
```suggestion
    response["subscriptions"] = plugin.subscriptions.values
        .map((subscription) => subscription.toMap())
        .toList();
    response["dynamic"] = plugin.dynamic;
```

Reply:
the `toMap` method here maybe it is not correct, you can just do `subscription.name`?